Source/WebKit/UIProcess/API/Cocoa/WKMenuItemIdentifiersPrivate.h

```diff
@@ -51,6 +51,7 @@ WK_EXTERN NSString * const _WKMenuItemIdentifierSearchWeb WK_API_AVAILABLE(macos
 WK_EXTERN NSString * const _WKMenuItemIdentifierShowHideMediaControls WK_API_AVAILABLE(macos(10.12), ios(10.0));
 WK_EXTERN NSString * const _WKMenuItemIdentifierShowHideMediaStats WK_API_AVAILABLE(macos(13.3), ios(16.4));
 WK_EXTERN NSString * const _WKMenuItemIdentifierToggleEnhancedFullScreen WK_API_AVAILABLE(macos(10.14), ios(12.0));
+WK_EXTERN NSString * const _WKMenuItemIdentifierToggleVideoViewer WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA));
```

Comment:
I wonder if these can be the actual versions now

Reply:
@pxlcoder do you know the answer to this?

Reply:
I think this is fine as-is. They will already resolve to the correct actual versions right now due to changes made in rdar://127507228.

And we can update all TBAs for the new release together.